src/client/entrypoint.cpp

```diff
@@ -55,6 +61,57 @@ ecs::World getGameWorld(ecs::Engine &engine)
     world.addSystem(ecs::systems::HandleIncomingMessages);
     world.addSystem(ecs::systems::SendDirection);
     world.addSystem(ecs::systems::movement);
+    world.addSystem(ecs::systems::HandleParallaxBounds);
+
+    ecs::Entity background1 = world.registry.spawn_entity();
+    ecs::Entity backgroundp1 = world.registry.spawn_entity();
+    ecs::Entity background2 = world.registry.spawn_entity();
+    ecs::Entity backgroundp2 = world.registry.spawn_entity();
+    ecs::Entity background3 = world.registry.spawn_entity();
+    ecs::Entity backgroundp3 = world.registry.spawn_entity();
+
+    world.registry.addComponent<ecs::component::Position>(background1, {0, 0});
```

Comment:
@aurelenc will confirm but their is a new way to load asset, anyway its working atm need to merge AssetLoader

Reply:
there*